src/reducers/commentReducer.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+import { CREATE_COMMENT, DELETE_COMMENT } from '../actions/commentActions';
+
+// -----------------------------------------------
+
+export const initialState = { 
+  comments: []
```

Comment:
since you are passing down the index with your post, you could use this the index to work with this state.